### DISCUSSION TITLE
How to make a backup?

### DISCUSSION THREAD

**Discussion by TangJames** [thijsvanloef/palworld-server-docker#272] [Q&A]
Short story is that I have the server running on a spare computer but I want to upgrade the hardware.

Can I just backup the palworld folder, do a new docker compose on my new computer, and copy/paste the files over?

**Comment by thijsvanloef** [comment]
You can create a backup using the `docker exec palworld-server backup` command. However you do need to manually import it on the different computer: https://github.com/thijsvanloef/palworld-server-docker?tab=readme-ov-file#manually-restore-from-a-backup
